faq.md

```diff
@@ -0,0 +1,36 @@
+# FAQ
+
+**What is Emissions API?**  
```

Comment:
yes, since I haven't made the formattion (table of content) I just used that for highlighting the questions. But yeah, using headline formattation is an option meanwhile.